Doofinder/Feed/Model/Data/InstallationStruct.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Doofinder\Feed\Model\Data;
+
+use JsonSerializable;
+
+class InstallationStruct implements JsonSerializable
+{
+    private $name;
```

Comment:
Try to include the PHPDocs:

```suggestion
    /**
     * @var string
     */
    private $name;
```

You can now validate it with `vendor/in/phpcs path/to/the/file.php` after having executed the composer install in the root folder.